packages/devtools_app/lib/src/screens/inspector/inspector_screen.dart

```diff
@@ -565,6 +569,59 @@ class _ErrorNavigatorButton extends StatelessWidget {
   }
 }
 
+class InspectorDetailsOption extends StatelessWidget {
+  const InspectorDetailsOption({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return ValueListenableBuilder(
+      valueListenable: preferences.inspector.defaultInspectorDetailsView,
+      builder: (context, selection, _) {
+        final theme = Theme.of(context);
+        return Column(
+          children: [
+            Text(
+              'Select the tab which opens by default when opening the inspector.',
+              style: theme.subtleTextStyle,
+            ),
+            const SizedBox(
+              height: denseSpacing,
+            ),
+            Row(
+              mainAxisSize: MainAxisSize.min,
+              children: [
+                Radio<InspectorDetailsViewOptions>(
+                  value: InspectorDetailsViewOptions.layoutExplorer,
+                  materialTapTargetSize: MaterialTapTargetSize.shrinkWrap,
+                  groupValue: selection,
+                  onChanged: _onChanged,
+                ),
+                const Text('Layout Explorer'),
```

Comment:
these strings should be added to the InspectorDetailsViewOptions enum, and should also be used for the tab titles so these values stay in sync